cmake/AutoRevision.txt

```diff
@@ -1,2 +1,2 @@
-8dcf1
-v3.0-dev
+9e418
```

Comment:
Don't update the commit, it's the branching point

Reply:
Oh didn't realize that. I thought it was latest id. I'll fix.